src/universe/math/geometry.py

```diff
@@ -253,12 +256,15 @@ def __init__(self, c: float = .0, m: float = 1.):
     def __str__(self):
         return f'y = {self.m}x + {self.c}'
 
+    def __repr__(self):
+        return self.__str__()
+
     @property
-    def norm_vec(self) -> Point2d:
+    def dir_vec(self) -> Point2d:
```

Comment:
Actually, this is a directional unit vector, since it's values are divided by the length of the line.

Reply:
I renamed it from norm_vec bacause it it not the normal vector. It is the normalized direction vector.

Reply:
Yes, and a normalized direction vector is indeed a unit vector. So, I suggest either adding a norm suffix (norm_dir_vec) or renaming it to unit_vec.

Reply:
Renamed dir_vec to unit_vec